### ISSUE TITLE
Error: @prisma/client did not initialize yet

### ISSUE THREAD

**Issue by flybayer** [blitz-js/blitz#1004]
### What is the problem?

If you delete node_modules, Prisma client isn't initialized when you run `blitz start`

### Workaround

1. Run `blitz db migrate`
2. Then run `blitz start`

### Steps to Reproduce

Inside a blitz app:
1. `rm -r node_modules`
2. `blitz start`
3. Try to load the page. -> `Internal Server Error`.

And then you get this in the console:

```
wait  - compiling...
event - compiled successfully
Error: @prisma/client did not initialize yet. Please run "prisma generate" and try to import it again.
In case this error is unexpected for you, please report it in https://github.com/prisma/prisma-client-js/issues/390.
    at new PrismaClient (/private/tmp/d3/node_modules/.prisma/client/index.js:3:11)
    at eval (webpack-internal:///./db/index.ts:12:50)
    at Module../db/index.ts (/private/tmp/d3/.blitz/caches/dev/.next/server/pages/_app.js:188:1)
    at __webpack_require__ (/private/tmp/d3/.blitz/caches/dev/.next/server/pages/_app.js:23:31)
    at eval (webpack-internal:///./app/auth/auth-utils.ts:9:60)
    at Module../app/auth/auth-utils.ts (/private/tmp/d3/.blitz/caches/dev/.next/server/pages/_app.js:116:1)
    at __webpack_require__ (/private/tmp/d3/.blitz/caches/dev/.next/server/pages/_app.js:23:31)
    at eval (webpack-internal:///./app/_resolvers/auth/mutations/login.ts:3:77)
    at Module../app/_resolvers/auth/mutations/login.ts (/private/tmp/d3/.blitz/caches/dev/.next/server/pages/_app.js:104:1)
    at __webpack_require__ (/private/tmp/d3/.blitz/caches/dev/.next/server/pages/_app.js:23:31)
```

### Versions

```
0.21.1
```

### Other

I confirmed that [runPrismaGeneration](https://github.com/blitz-js/blitz/blob/canary/packages/cli/src/commands/db.ts#L48) does actually run successfully. It is called on `blitz start` [here](https://github.com/blitz-js/blitz/blob/canary/packages/cli/src/commands/start.ts#L44).

So something weird going on here, possibly with the blitz compilation step. Or maybe some type of race condition.

**Comment by ntgussoni** [comment]
@flybayer  I'm having trouble running my dev environment so if anyone can take a quick look to see if this fixes it i'll make a proper PR. 

I couldn't find `runPrismaGeneration` being called on dev start, I know this code was here at some point so I wonder if it got removed by accident or i'm completely missing something

**Comment by flybayer** [comment]
This is no longer an issue since https://github.com/blitz-js/blitz/pull/1121

**Comment by bde-maze** [comment]
Hello there, just letting you now that I faced the same issue today (10th of November 2020) after running ```npm install``` in the project directory.

I was able to fix it using the same solution presented.

Blitz version:
```
You are using alpha software - if you have any problems, please open an issue here:
  https://github.com/blitz-js/blitz/issues/new/choose

macOS Catalina | darwin-x64 | Node: v14.5.0

blitz: 0.16.5-canary.5 (global)
blitz: 0.24.3 (local)
```

**Comment by flybayer** [comment]
@bde-maze update your global blitz version to latest, and that should resolve this for you

**Comment by Lokprakash-babu** [comment]
@flybayer I faced this issue today after running `yarn` in the project directory. 

OS: Windows 11
Node version: 16.13.2
Blitz version: 0.45.2
@prisma/client version: 3.9.1

**Comment by Lokprakash-babu** [comment]
@flybayer I faced this issue today after running `yarn` in the project directory. I tried with `npm install` faced the same issue. Unfortunately, `blitz db migrate` shows that the command is no longer available. So, I added a script in package.json for `prisma generate` and executed that before running `npm run dev` fixed the issue. 

OS: Windows 11
Node version: 16.13.2
Blitz version: 0.45.2
@prisma/client version: 3.9.1
